src/main/java/hubturbo/embeddedbrowser/jxbrowser/JxBrowserAdapter.java

```diff
@@ -79,12 +90,18 @@ public void executeCommand(int command) {
     }
 
     @Override
-    public void addLoadListener(EbLoadListener listener) {
-        this.listener = listener;
+    public void setLoadListener(EbLoadListener listener) {
+        this.loadListener = Optional.ofNullable(listener);
+        this.browserView.getBrowser().removeLoadListener(this);
```

Comment:
setLoadListener may get called multiple times. I'm afraid that it adding the same instance twice may result in calling it twice. The documentation didn't state the details. 

I will remove the removeLoadListener and observe if it will get called multiple times if the listener is added multiple times.